docs/0.4/concepts/splinter_registry.md

```diff
@@ -0,0 +1,227 @@
+metadata key is set for a node in multiple registries, precedence is determined
+in the same way as node definitions.
+
+### Recommended Patterns
+
+External registries are the recommended way to manage lists of known Splinter
+nodes because they can be shared in a consistent way across a Splinter network.
+The internal registry is only available on the local Splinter node, so any data
+added, modified, or removed from it will not be reflected on other nodes. This
+can be useful in some cases, but generally it's desirable for node data to be
+consistent across the network.
+
+While most node data is static, it is sometimes necessary to modify the
+properties of a node that's defined in an external registry. This can be done
+using the internal registry, but it's usually better to modify the external
+source registry directly.
+
+If an external registry is modified (the file is edited or the database is
+updated), the unified registry will detect the changes and update the
+information it provides. How upstream changes are detected and loaded by the
+unified registry depends on the type of source registry.
+
+### Types of Source Registries
+
+There are currently two source registry implementations provided by Splinter:
+
+* **Local YAML files** exist on the filesystem. The local file is cached in
+  memory, so if the file is removed or becomes unavailable, its contents will
+  remain available as long as the Splinter node is running. Any changes made
+  directly to the file will be immediately detected by the unified registry.
+
+* **Remote YAML files** are accessed over HTTP and cached to a local YAML file;
+  if the remote file becomes unavailable, its contents from the last successful
+  read will continue to be available, even across restarts. Changes made to
+  remote files are detected by the unified registry in two ways:
+
+    * **Automatic refresh**: The remote file is fetched at a regular interval of
+      time
+
+    * **Forced refresh**: After a period of time since the last refresh, the
+      next read operation will fetch the remote file; this is similar to cache
+      invalidation.
+
+Local and remote YAML files contain arrays of nodes, where each node definition
+provides all required fields and some optional metadata:
+
+``` yaml
+- identity: "Node1"                       # Required, must not be empty
+  endpoints:                              # At least one endpoint is required
+    - "tcps://123.0.0.123:8080"           # Must not be empty
+  display_name: "Bitwise IO - Node 1"     # Required, must not be empty
+  keys:                                   # At least one key is required
+    - "000000000000000000000000000000000" # Must not be empty
+  metadata:                               # May be empty
+    company: "Cargill"
+...
+```
+
+## Configuring the splinterd Registry
+
+The external source registries that will be used by splinterd are configured
```

Comment:
Change "external source registries" to "external registries" or "external sources".

Reply:
Dropped "source"